compiler/codegen/src/compile.rs

```diff
@@ -2903,9 +2903,15 @@ impl Compiler {
         for feature in features {
             match feature.name.as_str() {
                 // Python 3 features; we've already implemented them by default
-                "nested_scopes" | "generators" | "division" | "absolute_import"
-                | "with_statement" | "print_function" | "unicode_literals" => {}
-                // "generator_stop" => {}
+                "nested_scopes"
+                | "generators"
+                | "division"
+                | "absolute_import"
+                | "with_statement"
+                | "print_function"
+                | "unicode_literals"
+                | "compile_future_features" => {}
+                "generator_stop" => {}
```

Comment:
Does this work?

```suggestion
                | "compile_future_features"
                | "generator_stop" => {}
```

Reply:
Oops, my autoformatter led to that silly mistake.

To answer your question: it seems to work.

I did a quick check on the Python interpreter and the import succeeds. The module also looks okay via `dir()` inspection.

Anyhow, I will fix it. Thanks for pointing that out.